public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -351,7 +350,6 @@ export class PrometheusDatasource
           ) {
             const exemplarTarget = cloneDeep(target);
             exemplarTarget.instant = false;
-            exemplarTarget.requestId += '_exemplar';
```

Comment:
This may be losing something interesting... but only if people are currently able to use and understand the exemplar usage in the current usage tracking (I am skeptical of that)